recipes/mikecore/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "mikecore" %}
+{% set version = "0.1.2b2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/75/88/4176a60a591a08f8473e24b1ba1bc7bf5bc14620290029af419c10b13586/mikecore-{{ version }}.tar.gz
+  sha256: b54c1639aad7607a6c15b6b964376fde4d31f82145a4b0e566d49890a7b797de
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
It looks like the build is just copying some pre-built .so/.dll files into the directory? That needs to be changed to build from source using conda-forge's compilers and library dependencies.

Reply:
Is it really necessary?

This package is similar to https://anaconda.org/conda-forge/mikeio and in this case, the binary dependencies come along with the PyPI package.

Reply:
It really is, and the `mikeio` package should really be doing the same (I think it just wasn't noticed during review of that recipe). Building everything helps ensure that different packages are compatible with regards to compiler, library symbols, etc.

Also running random binaries from somewhere on the internet is a huge security no-no, and there can also be licensing issues depending on what else is linked (e.g. statically) inside those binaries.

Reply:
We only repackage binaries as the absolute last resort, in extreme circumstances.